### PR TITLE
kup-accordion: highlight string searched with globalFilter

### PR DESCRIPTION
**Pull request by chiarazambe-smeup**

### CODE REVIEW

File: packages/ketchup/src/assets/accordion.js [hunk 1]
```diff
@@ -194,6 +194,7 @@ const ksData = {
 const ksAccordion = document.getElementById('ks');
 ksAccordion.data = ksData;
 ksAccordion.globalFilter = 'true';
+ksAccordion.globalFilterMode = 'HIGHTLIGHT';
```
Comment (stefanomacconi) [inline packages/ketchup/src/assets/accordion.js:197]: Correggere con highlight

File: packages/ketchup/src/components/kup-accordion/kup-accordion.tsx [hunk 2]
```diff
@@ -412,6 +429,47 @@ export class KupAccordion {
         }
     }
 
+    /**
+     * Renders a content with a part highlighted.
+     * NOTE: same in kup-accordion and in kup-tree
+     */
+    private renderHighlightedContent(
+        content: string,
+        hightligh: string,
```
Comment (stefanomacconi) [inline packages/ketchup/src/components/kup-accordion/kup-accordion.tsx:438]: Typo: same as above

File: packages/ketchup/src/components/kup-accordion/styles/kup-accordion-main.scss [hunk 3]
```diff
@@ -72,6 +72,11 @@
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: pre;
+
+      &--highlighted {
```
Comment (stefanomacconi) [inline packages/ketchup/src/components/kup-accordion/styles/kup-accordion-main.scss:76]: Must use the themes

File: packages/ketchup/src/utils/filters/filters-declarations.ts [hunk 4]
```diff
@@ -35,3 +35,8 @@ export interface Filter {
  * @property {string} 5 - Always equal to ': it's the end of the filter.
  */
 export const FILTER_ANALIZER = /^('|!')(%){0,1}(.*?)(%){0,1}(')$/;
+
+export enum GlobalFilterMode {
```
Comment (stefanomacconi) [inline packages/ketchup/src/utils/filters/filters-declarations.ts:39]: Replace value "SIMPLE" with "simple".
Same for "highlight"

**Review by foscili-smeup — APPROVED** [review]

**Pull request merged**